docs/earthfile/earthfile.md

```diff
@@ -677,6 +677,13 @@ SAVE IMAGE my-example-registry.com/another-image:latest
 ```
 {% endhint %}
 
+{% hint style='danger' %}
+##### Important
+
+As of [`VERSION 0.6`](#version), the `--referenced-save-only` feature flag is enabled by default. Images are only saved if they are connected to the initial target through a chain of `BUILD` commands. [See the `--referenced-save-only` flag documentation for more details](./features.md#referenced-save-only).
```

Comment:
Instead of the feature flag link here, I would link to BUILD and point to "What is output and pushed".

Reply:
Why not both

Reply:
I'm trying to avoid confusing new users with the term "feature flag" :-)

Just thinking that simpler is better - they don't need to know about our history to get going.

Reply:
Friendly ping on this. I know it seems innocuous, but the extra simplicity is worth it.